server/tournaments/index.ts

```diff
@@ -1263,6 +1287,30 @@ function createTournament(
 }
 
 const commands: Chat.ChatCommands = {
+	pasttours: 'recenttours',
+	recenttours(target, room, user) {
+		this.runBroadcast();
+		room = this.requireRoom();
+		if (!room.settings.tournaments?.recentToursLength) {
+			throw new Chat.ErrorMessage(`Recent tournaments aren't documented in this room.`);
+		}
+		if (!room.settings.tournaments?.recentTours?.length) {
+			throw new Chat.ErrorMessage(`There haven't been any documented tournaments in this room recently.`);
+		}
+		// Shorten array if the recentToursLength gets adjusted
+		const array = room.settings.tournaments.recentTours.filter((x, i) => (
+			i <= room!.settings.tournaments!.recentToursLength! - 1
+		));
+		const {name, time} = array[0];
+		let buf = `The last tournament ended ${Chat.toDurationString(Date.now() - time)} ago - ${name}`;
+		if (array.length > 1) {
+			buf += `<hr /><strong>Previous tournments:</strong> `;
+			buf += array.filter((x, i) => i !== 0).map(x => x.name).join(', ');
+		}
+		this.sendReplyBox(buf);
+	},
+	recenttourshelp: [`/recenttours - Displays the n most recent tour(s).`],
```

Comment:
n?

Reply:
variable n as in n, n-1, n-2, i couldnt figure out a better way to phrase this while remaining compact

Reply:
You never mention `n` anywhere else in the help, though, so it's unclear what it refers to. It seems like it just displays however many recent tournaments have been recorded (which will be at minimum 0 and at most `settings.tournaments.recentTourLength`).